openshift/capabilities-tool-job.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: capabilities-tool-job
+  name: opcap-job
 spec:
   template:
     spec:
       containers:
-      - name: capabilities-tool
-        image: quay.io/opdev/capabilities-tool:v0.2.7
-        args: ["index","capabilities","--container-engine","podman","--output-path","/opt/capabilities-tool","--bundle-image","$(bundle_image)","--bucket-name","$(bucket_name)","--bundle-name","$(bundle_name)"]
+      - name: opcap
+        image: quay.io/opdev/opcap:v4.9
```

Comment:
Is 4.9 to align with OpenShift? I think we should keep versions on 0.x.y until we get this actually in OpenShift. That really suggests two things one that we are in stable stream or that we are already part of OCP release. Normally we would have an upstream project with a proper version and a downstream to be distributed with OCP if we will. The downstream one would be the tool with OCP numbers. Another thought is that we could tie the orchestrator/runner/operator to OCP versions but allow opcap to evolve independently and just set the supported versions of opcap on the orchestration tool according to what has been tested.

Reply:
@mrhillsman

Reply:
I don't think our tag specifically speaks to being part of OpenShift but I understand the stable stream argument. I like the latter suggestion @acmenezes makes. We want to tie the overall audit to a version of OpenShift as it is what we are stating to have successfully run the tooling against and any audit of said tooling against a version of OpenShift that it does not match has not been tested. That being said I think the runner and eventually the operator is the appropriate place to put the tags.

Reply:
>Another thought is that we could tie the orchestrator/runner/operator to OCP versions but allow opcap to evolve independently and just set the supported versions of opcap on the orchestration tool according to what has been tested.


@acmenezes Makes sense, I'm going to change the opcap version to 0.0.1.